portals/publisher/src/main/webapp/source/src/app/components/Apis/Create/CreateAPIWithAI/components/ApiChatExecute.tsx

```diff
@@ -281,6 +285,7 @@ const ApiChatExecute: React.FC<ApiChatExecuteProps> = ({
                             onKeyDown={handleKeyDown}
                             testId='nl-query-input'
                             multiline
+                            disabled={(designAssistantEnabled && !aiAuthTokenProvided)||!designAssistantEnabled}
```

Comment:
this can simplify to (!aiAuthTokenProvided || !designAssistantEnabled) isn't it?